documentation/docs/02_account_manager.md

```diff
@@ -6,59 +9,93 @@ The account manager interface allows you to:
 
 # Commands
 
-## `help`
+## `./wallet`
 
-### Parameters
+Runs the wallet without a specified account:
+- If the wallet has only one account, it will be used;
+- If the wallet has more than one account, a selector will be shown to decide which account to use.
+
+The wallet needs to be initialised (`init` command) and with at least one account (`new` command).
```

Comment:
Do you think it should be like this? I guess we could also change the `init` to create a new account directly, not sure if that would be too much or better

Reply:
Tbh I think it's not unreasonable to have them be 2 separate commands